packages/react-components/src/components/InputBoxComponent.test.tsx

```diff
@@ -267,13 +279,21 @@ describe('InputBoxComponent should hide mention popover', () => {
 
   const checkSuggestionsNotShown = (): void => {
     for (let i = 0; i < suggestions.length; i++) {
-      const contextMenuItem = screen.queryByText(suggestions[i].displayText);
+      const suggestionText = suggestions[i]?.displayText;
+      if (!suggestionText) {
+        throw new Error('Suggestion text is not defined');
+      }
+      const contextMenuItem = screen.queryByText(suggestionText);
       expect(contextMenuItem).toBeNull();
     }
   };
 
   const checkSuggestionsShown = async (): Promise<HTMLElement> => {
-    const firstSuggestionMenuItem = await screen.findByText(suggestions[0].displayText);
+    const firstSuggestionText = suggestions[0]?.displayText;
+    if (!firstSuggestionText) {
+      throw new Error('Suggestion text is not defined');
+    }
+    const firstSuggestionMenuItem = await screen.findByText(firstSuggestionText);
```

Comment:
Feel like we should have these lines into a separate function since they've been repeated quite a bit

Reply:
Simplified with foreach instead